pkg/apis/k8s.cni.cncf.io/v1/types.go

```diff
@@ -106,6 +106,7 @@ type NetworkStatus struct {
 	Default    bool        `json:"default,omitempty"`
 	DNS        DNS         `json:"dns,omitempty"`
 	DeviceInfo *DeviceInfo `json:"device-info,omitempty"`
+	Gateway    string      `json:"gateway,omitempty"`
```

Comment:
Actually, this needs to be a list, I missed that.

Reply:
Done.